README.md

```diff
@@ -72,20 +72,21 @@ Where 16 is the archetype version number that you want to use (see archetype ver
 
 ### Available properties
 
-Name               | Description
--------------------|------------------------------
-groupId            | Base Maven groupId
-artifactId         | Base Maven ArtifactId
-version            | Version
-package            | Java Source Package
-appsFolderName     | /apps folder name
-artifactName       | Maven Project Name
-componentGroupName | AEM component group name
-contentFolderName  | /content folder name
-confFolderName     | /conf folder name
-cssId              | prefix used in generated css
-packageGroup       | Content Package Group name
-siteName           | AEM site name
+Name                 | Default | Description
+---------------------|------------------------------
+groupId              |         | Base Maven groupId
+artifactId           |         | Base Maven ArtifactId
+version              |         | Version
+package              |         | Java Source Package
+appsFolderName       |         | /apps folder name
+artifactName         |         | Maven Project Name
+componentGroupName   |         | AEM component group name
+contentFolderName    |         | /content folder name
+confFolderName       |         | /conf folder name
+cssId                |         | prefix used in generated css
+packageGroup         |         | Content Package Group name
+siteName             |         | AEM site name
+optionIncludeExample |    y    | Include Component Library example
```

Comment:
would make this `optionIncludeExamples` for future proofing, this package will have more example sites down the line for e.g. templates.

Reply:
@richardhand Thanks for your feedback. Changed the name as you suggested with my latest commit